tests/Unit/Evolution/Systems/Cce/Test_OptionTags.cpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/Systems/Cce/OptionTags.hpp"
+#include "Evolution/Systems/Cce/ReadBoundaryDataH5.hpp"
+#include "NumericalAlgorithms/Interpolation/CubicSpanInterpolator.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "tests/Unit/Evolution/Systems/Cce/BoundaryTestHelpers.hpp"
+#include "tests/Unit/TestCreation.hpp"
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.OptionTags",
+                  "[Unit][Evolution]") {
+  CHECK(TestHelpers::test_creation<size_t, Cce::OptionTags::LMax>("8") == 8u);
+  CHECK(TestHelpers::test_creation<size_t, Cce::OptionTags::ObservationLMax>(
+            "6") == 6u);
+  CHECK(
+      TestHelpers::test_creation<size_t, Cce::OptionTags::NumberOfRadialPoints>(
+          "3") == 3u);
+  CHECK(TestHelpers::test_creation<double, Cce::OptionTags::EndTime>("4.0") ==
+        4.0);
+  CHECK(TestHelpers::test_creation<double, Cce::OptionTags::StartTime>("2.0") ==
+        2.0);
+  CHECK(TestHelpers::test_creation<double, Cce::OptionTags::TargetStepSize>(
+            "0.5") == 0.5);
+  CHECK(TestHelpers::test_creation<std::string,
+                                   Cce::OptionTags::BoundaryDataFilename>(
+            "CceR0100.h5") == "CceR0100.h5");
+  CHECK(TestHelpers::test_creation<size_t, Cce::OptionTags::H5LookaheadTimes>(
+            "5") == 5u);
+  CHECK(TestHelpers::test_creation<size_t,
+                                   Cce::OptionTags::ScriInterpolationOrder>(
+            "4") == 4u);
+  const std::string filename = "CceR0100.h5";
```

Comment:
have the name of the file include the test name so we don't get conflicts when running tests in parallel